src/components/ManaFilter.js

```diff
@@ -27,38 +27,36 @@ export default function ColorFilter(props) {
 
   return (
     <>
-      <div className="flex flex-row justify-between my-4 mx-0 max-w-4/5 w-96">
+      <div className="flex flex-row my-4 mx-0">
```

Comment:
Curious about the removal of the max-width. Without it, and the mx-0, the buttons go almost all the way to the edge of the screen. I suppose you could accomplish some breathing room with a bit of margin. 

There is also a bit of horizontal scroll on my device (iPhone 11 Pro Max)

Reply:
```suggestion
      <div className="flex flex-row my-4 mx-1">
```

`mx-1` will help this here, without the need for a max width.